validation/multi_region/multi_region_tracer_advection.jl

```diff
@@ -87,31 +154,28 @@ simulation.callbacks[:save_tracer] = Callback(save_tracer, TimeInterval(Δt))
 
 run!(simulation)
 
+
 @info "Making an animation from the saved data..."
 
-plot_type = :heatlatlon # Choose plot_type to be :heatsphere or :heatlatlon.
+n = Observable(1)
 
-fig = Figure(resolution = (850, 750))
-title = "Tracer Concentration"
+Θₙ = []
+for region in 1:6
+    push!(Θₙ, @lift interior(getregion(tracer_fields[$n], region), :, :, grid.Nz))
+end
+
+fig = Figure(resolution = (2000, 400))
 
-if plot_type == :heatsphere
-    ax = Axis3(fig[1,1]; xticklabelsize = 17.5, yticklabelsize = 17.5, title = title, titlesize = 27.5, titlegap = 15, 
-               titlefont = :bold, aspect = (1,1,1))
-    heatsphere!(ax, tracer_fields[1]; colorrange = (-1, 1))
-elseif plot_type == :heatlatlon
-    ax = Axis(fig[1,1]; xticklabelsize = 17.5, yticklabelsize = 17.5, title = title, titlesize = 27.5, titlegap = 15, 
-              titlefont = :bold)
-    heatlatlon!(ax, tracer_fields[1]; colorrange = (-1, 1))
+for region in 1:6
+    ax = Axis(fig[1, region], title="panel $region")
+    heatmap!(ax, Θₙ[region], colorrange=(0, θ₀))
 end
 
+fig
+
 frames = 1:length(tracer_fields)
 
-GLMakie.record(fig, "multi_region_tracer_advection.mp4", frames, framerate = 1) do i
-    msg = string("Plotting frame ", i, " of ", frames[end])
-    print(msg * " \r")
-    if plot_type == :heatsphere
-        heatsphere!(ax, tracer_fields[i]; colorrange = (-1, 1), colormap = :balance)
-    elseif plot_type == :heatlatlon
-        heatlatlon!(ax, tracer_fields[i]; colorrange = (-1, 1), colormap = :balance)
-    end
-end
```

Comment:
@siddharthabishnu this was making another plot for every frame. What I did just changes the array plotted and it's much much much faster.

Reply:
I see. The plotting is indeed much faster now. Thanks.